### ISSUE TITLE
elfet/php-ssh Not found

### ISSUE THREAD

**Issue by sidz** [deployphp/deployer#158]
Hi,
Please fix composer.json require section in the Deployer 2. It use old repo "elfet/php-ssh", that already droped.

**Comment by elfet** [comment]
Yeah, sorry about that.

**Comment by elfet** [comment]
Fixed.

**Comment by tomaj** [comment]
Hi,
It looks like there is still some issue with this. After i updated composer requirements to ```"deployphp/deployer": "2.0.4"```

it shows this error: 
```
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - deployer/deployer v2.0.4 requires phpseclib/phpseclib 2.0.*@dev -> satisfiable by phpseclib/phpseclib[2.0.x-dev].
    - deployer/deployer v2.0.4 requires phpseclib/phpseclib 2.0.*@dev -> satisfiable by phpseclib/phpseclib[2.0.x-dev].
    - Removal request for phpseclib/phpseclib == 2.0.9999999.9999999-dev
    - Installation request for deployer/deployer 2.0.4 -> satisfiable by deployer/deployer[v2.0.4].
```

When i put this in my composer.json it works fine:
```"phpseclib/phpseclib": "2.0.*@dev"```

Is this requirement to explicitly put *phpseclib* dependency to composer?

**Comment by elfet** [comment]
Thus is issue with phpseclib and packagist. I already created issue there.

**Comment by elfet** [comment]
Fixed.

**Comment by swemaniac** [comment]
I'm having the same problem whatever version I try to require

**Comment by elfet** [comment]
Yeeahh. Fix is coming....

**Comment by swemaniac** [comment]
:)

**Comment by elfet** [comment]
I know how to hotfix...

**Comment by elfet** [comment]
Will be fixed in #182
